scripts/upgrade

```diff
@@ -101,42 +97,90 @@ then
   if 	[ -d "$backup_squelettes" ]
   then
     cp -ar $backup_squelettes $final_path/www/squelettes
-    ynh_secure_remove  $backup_squelettes
   fi
 
   if [ -e "$backup_bdd" ]
   then
 	  cp -a $backup_bdd $final_path/association.sqlite
-    ynh_secure_remove $backup_bdd
   fi
 
   chown -R $app:$app $final_path
+  chmod 755 $final_path
+
+  # Create the visitors permission if needed
+  visitors_enabled=$(ynh_permission_has_user "main" "visitors" && echo 0 || echo 1)
+  if [[ $visitors_enabled -eq 1 ]]; then
+    ynh_permission_update --permission "main" --add "visitors"
+    ynh_systemd_action --service_name=nginx --action=reload
+    # Finalize the upgrade
+    ynh_local_curl "/index.php"
+    sleep 5
+    ynh_local_curl "/index.php"
+    upgrade_type="UPGRADE_APP"
+    ynh_permission_update --permission "main" --remove "visitors"
+  fi
+
+  if [[ $visitors_enabled -eq 0 ]]; then
+    ynh_systemd_action --service_name=nginx --action=reload
+    # Finalize the upgrade
+    ynh_local_curl "/index.php"
+    sleep 5
+    ynh_local_curl "/index.php"
+    upgrade_type="UPGRADE_APP"
+  fi
```

Comment:
```suggestion
  # Temporarily enable visitors for curl if needed
  visitors_enabled=$(ynh_permission_has_user "main" "visitors" && echo yes || echo no)
  if [[ $visitors_enabled == "no" ]]; then
    ynh_permission_update --permission "main" --add "visitors"
  fi  

  # Finalize the upgrade
  ynh_local_curl "/index.php"
  sleep 5
  ynh_local_curl "/index.php"
  
  if [[ $visitors_enabled == "no" ]]; then
    ynh_permission_update --permission "main" --remove "visitors"
  fi
```

I guess the current version works but we can improve the readability / avoid duplication ... also I doubt that there's really a need to reload nginx, nor to force the value of `upgrade_type` (it should already be set to `UPGRADE_APP` at the beginning of the script i believe)